Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps.py

```diff
@@ -0,0 +1,550 @@
+    md = tableToMarkdown('Hashes status:', ids_obj, headerTransform=pascalToSpace)
+    context = {'Traps.File(val.SHA256 == obj.SHA256)': ids_obj}
+    return_outputs(md, context)
+
+
+def endpoint_isolate_command():
+    args = demisto.args()
+    endpoint_id = args.get('endpoint_id')
+    operation_obj = endpoint_isolate(endpoint_id)
+    md = tableToMarkdown(f'Isolate command on endpoint {endpoint_id} received', operation_obj,
+                         headerTransform=pascalToSpace)
+    context = {'Traps.Isolate(val.OperationID == obj.OperationID)': operation_obj}
+    return_outputs(md, context)
+
+
+def endpoint_isolate_status_command():
+    args = demisto.args()
+    operation_id = args.get('operation_id')
+    isolate_status = endpoint_isolate_status(operation_id)
+    md = f'### Isolate status is: {isolate_status.get("Status")}'
+    context = {f'Traps.IsolateResult(val.OperationID == obj.OperationID)': isolate_status}
+    return_outputs(md, context)
+
+
+def main():
+    # Remove proxy if not set to true in params
+    handle_proxy()
+
+    ''' COMMANDS MANAGER / SWITCH PANEL '''
+
+    LOG('Command being called is %s' % (demisto.command()))
+
+    try:
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration test button.
+            test_module_command()
+            demisto.results('ok')
+        elif demisto.command() == 'traps-get-endpoint-by-id':
+            get_endpoint_by_id_command()
+        elif demisto.command() == 'traps-endpoint-files-retrieve':
+            endpoint_files_retrieve_command()
+        elif demisto.command() == 'traps-endpoint-files-retrieve-result':
+            endpoint_files_retrieve_result_command()
+        elif demisto.command() == 'traps-endpoint-scan':
+            endpoint_scan_command()
+        elif demisto.command() == 'traps-endpoint-scan-result':
+            endpoint_scan_result_command()
+        elif demisto.command() == 'traps-event-update':
+            event_update_command()
+        elif demisto.command() == 'traps-event-bulk-update-status':
+            event_bulk_update_status_command()
+        elif demisto.command() == 'traps-hash-blacklist':
+            hash_blacklist_command()
+        elif demisto.command() == 'traps-hash-blacklist-remove':
+            hash_blacklist_remove_command()
+        elif demisto.command() == 'traps-hashes-blacklist-status':
+            hashes_blacklist_status_command()
+        elif demisto.command() == 'traps-event-quarantine':
+            event_quarantine_command()
+        elif demisto.command() == 'traps-event-quarantine-result':
+            event_quarantine_result_command()
+        elif demisto.command() == 'traps-endpoint-isolate':
+            endpoint_isolate_command()
+        elif demisto.command() == 'traps-endpoint-isolate-status':
+            endpoint_isolate_status_command()
+    # Log exceptions
+    except Exception as e:
+        LOG(e)
+        LOG.print_log()
```

Comment:
remove LOG(e) and LOG.print_log() `return_error` do them both

Reply:
- [x] Fixed